specification/translation/data-plane/Azure.AI.DocumentTranslation/stable/2024-05-01/openapi.json

```diff
@@ -728,17 +726,12 @@
       "description": "Document Translate Request Content",
       "properties": {
         "document": {
-          "type": "string",
-          "format": "byte",
+          "type": "object",
```

Comment:
the reason for those kind of changes is that now this model is an unreachable model where the httppart is now an empty object